content/shpclmove.tex

```diff
@@ -1,5 +1,5 @@
 \apisummary{
-    Extends  a symmetric heap block or copies the contents of the block into a
+    Extends  a symmetric heap block, or copies the contents of the block into a
```

Comment:
Sure, this is a sentence fragment to begin with, however, see _Comma Abuse No. 13_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).